Gems/Atom/Feature/Common/Code/Source/PostProcess/ChromaticAberration/ChromaticAberrationSettings.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Math/MathUtils.h>
```

Comment:
Not sure if these are needed by the inl, but could you do a quick reorder/cleanup of the includes?
Could also do that on all the other files?

Reply:
Took at quick look and did some reordering in latest commit